manuscript.md

```diff
@@ -1,8 +1,33 @@
-# Meta
+---
+bibliography: [references.bib]
+---
 
 **BOX 1: Biological Examples**
 
-## What is the paper about?
+## Why should we predict and forecast species interaction networks?
+
+Networks of species interactionss underpin our understanding of key ecological
+processes [@Pascual2006EcoNet; @Heleno2014EcoNet]. Although they have initially
+be used to describe the interactions *within* a community, our interest in the
+last decade has shifted towards understanding their structure and their
```

Comment:
Actually I wonder if we should expand on *why* network structure is just as (if not more) informative/important as interactions. It might come up at another point in the ms but maybe a good idea to raise earlier rather than later?

Reply:
It will definitely come up in the part on network variation - let's leave it at that for now, to keep the answer short.